DOCSP-28410: Add link from SDK Model pages to App Services schema mapping page (#2958)
mongodb/docs-realm: source/includes/map-to-bson-type.rst

> To map a specific data type to a BSON data type in an 

Reviewer: This is a nitpick, but the linked page isn't really telling users how to map a specific data type to a BSON data type. It's more an explanation of how the mapping works (and not a great one, at that, I see after re-reading the page). Maybe we can tweak this wording to better set expectations?
```suggestion
To learn how specific data types are mapped to BSON types in an 
```

Author: Agreed. Thanks!
